FinSearchUnified/tests/PluginTest.php

```diff
@@ -31,4 +39,191 @@ public function testCalculateGroupkey()
         $this->assertEquals($usergroup, $decrypted);
 
     }
+
+    /**
+     * @param $number
+     * @param $isActive
+     * @return \Shopware\Models\Article\Article
+     * @throws \Shopware\Components\Api\Exception\CustomValidationException
+     * @throws \Shopware\Components\Api\Exception\ValidationException
+     */
+    private function createTestProduct($number, $isActive)
+    {
+        // create temporary test product for the export cases
+        $testArticle = array(
+            'name' => 'TestArticle' . $number,
+            'active' => $isActive,
+            'tax' => 19,
+            'supplier' => 'Test Supplier',
+
+            'categories' => array(
+                array('id' => 3),
+                array('id' => 5),
+            ),
+            'images' => array(
+                array('link' => 'https://via.placeholder.com/300/F00/fff.png'),
+                array('link' => 'https://via.placeholder.com/300/09f/000.png'),
+            ),
+
+            'mainDetail' => array(
+                'number' => 'SW100' . $number,
+                'active' => $isActive,
+                'inStock' => 16,
+                'prices' => array(
+                    array(
+                        'customerGroupKey' => 'EK',
+                        'price' => 99.34,
+                    ),
+                )
+            ),
+        );
+        $manger = new \Shopware\Components\Api\Manager();
+        $resource = $manger->getResource('Article');
+        $article = $resource->create($testArticle);
+        $this->assertInstanceOf(\Shopware\Models\Article\Article::class, $article);
+
+        return $article;
+    }
+
+    /**
+     * @throws \Shopware\Components\Api\Exception\CustomValidationException
+     * @throws \Shopware\Components\Api\Exception\ValidationException
+     */
+    public function testBothActiveProductExported()
+    {
+
```

Comment:
Please go through all methods and make sure that there are no empty lines after `{` and before `}`.